### PR TITLE
Bump zfs

### PR DESCRIPTION
**Pull request by jgrund**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,25 +7,43 @@ jobs:
...
               - docker run -d -it --name libzfs -v $(pwd):/rust-libzfs:rw imlteam/zfs
+              - docker exec -i libzfs bash -c 'yum install -y cargo yum-plugin-copr'
+              - docker exec -i libzfs bash -c 'yum copr -y enable alonid/llvm-5.0.0'
+              - docker exec -i libzfs bash -c 'yum install -y clang-5.0.0'
               - docker exec -i libzfs bash -c "cd /rust-libzfs/libzfs-sys && cargo package && cargo publish --token $CARGO_TOKEN"
         - stage: deploy-libzfs
           name: "libzfs"
           script:
               - docker run -d -it --name libzfs -v $(pwd):/rust-libzfs:rw imlteam/zfs
+              - docker run -d -it --name libzfs -v $(pwd):/rust-libzfs:rw imlteam/zfs
```
Comment (utopiabound) [inline .travis.yml]: Duplicate line?

File: Vagrantfile [hunk 2]
```diff
@@ -55,20 +56,17 @@ Vagrant.configure('2') do |config|
...
   config.vm.provision 'shell', inline: <<-SHELL
-    yum -y install yum-plugin-copr epel-release http://download.zfsonlinux.org/epel/zfs-release.el7_5.noarch.rpm
+    yum -y install yum-plugin-copr epel-release http://download.zfsonlinux.org/epel/zfs-release.el7_6.noarch.rpm
+    yum-config-manager --disable zfs
+    yum-config-manager --enable zfs-kmod
     yum -y copr enable alonid/llvm-5.0.0
-    yum -y install clang-5.0.0 zfs libzfs2-devel --nogpgcheck
-    modprobe zfs
+    yum -y install clang-5.0.0 zfs libzfs2-devel cargo --nogpgcheck
     genhostid
```
Comment (utopiabound) [inline Vagrantfile]: This may want to be `zgenhostid` (provided by zfs instead of built-in from rhel)

File: .travis.yml (hunk 1 above)
Comment (jgrund) [reply .travis.yml]: yes

File: Vagrantfile (hunk 2 above)
Comment (jgrund) [reply Vagrantfile]: Ok, I'll try it

**Pull request merged**